walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,23 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
+    Assert.assertEquals("Hello Ada", greeting);
```

Comment:
nit: make this a static import so it can be used as "assertEquals".

Reply:
fixed